editor/editor_themes.cpp

```diff
@@ -37,8 +37,8 @@
 #include "editor_settings.h"
 
 #include "modules/modules_enabled.gen.h"
-#ifdef MODULE_SVG_ENABLED
-#include "modules/svg/image_loader_svg.h"
+#ifdef MODULE_THORVG_ENABLED
+#include "modules/thorvg/image_loader_thorvg.h"
```

Comment:
I suggest keeping the `modules/svg/image_loader_svg.h` name, only the thirdparty library needs to change (nanosvg -> thorvg).

That avoids having to do all those renames and the library change is transparent to the rest of the engine.